### PR TITLE
Fixing dates for 2021

### PR DESCRIPTION
**Pull request by emarcey**

### CODE REVIEW

**Comment by ubershmekel** [comment]
@emarcey these should all say 2020, not 2021, no?

File: tools/data_builder.py [hunk 1]
```diff
@@ -57,9 +57,16 @@ def title_to_name_date(line):
     if date_text in ("Date Unknown", "Unknown Date"):
         return name, "", "Unknown Date"
 
-    date_found = date_regex.search(date_text).group()
+    date_search = date_regex.search(date_text)
+    date_found = date_text.lower().replace("(believed to be)", "").strip()
+    if date_search:
+        date_found = date_search.group()
+
     date = parse(date_found).strftime("%Y-%m-%d")
```
Comment (ubershmekel) [inline tools/data_builder.py:65]: I would do something like `if '202' not in date_found: date_found += ' 2020'`.

Comment (emarcey) [reply tools/data_builder.py:65]: Yeah I tried that and it didn't really work but I'll tinker again because I royally fucked some of this up 🤦‍♀️

Comment (emarcey) [reply tools/data_builder.py:67]: Okay I think I unfucked it

**Review by ubershmekel — APPROVED** [review]

**Pull request merged**